_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+The basics are Hardware Inputs, Digital Inputs, and Hardware Outputs. You will likely notice that I will be refraining from going into depth on individual features in this article and mostly want to give a sum of the basics to get started. The extra features I would like to cover in a different article altogether.
+
+Lets go over what each of the basic pieces are. Hardware Inputs are typically something like a microphone, line input, or similar things. The Virtual Input is basically a digital audio device/cable. (I will get more into these in one of the setups and further explain them.) Lastly, we have Hardware Ouputs, these are things like headphones, speakers, or in the case of what we will be configuring for streaming in the setup part, a digital audio device/cable.
+
+[picture of a fully setup voicemeeter maybe?]
+
+All of these options can be confusing when first looking at them so what I will do is break down each portion per version of voicemeeter. The portions covered at this moment will be Hardware Outputs, then Digital Inputs, then Hardware Inputs. Why does it seem like I'll be going backwards from what I originally listed? Unlike what the program looks like, which is the order up above, this new order is based on being able to check to make sure everything is getting set up properly as you go. You need your outputs such as headphones for testing audio to make sure the inputs are set up properly. If we set all our inputs up and then our outputs, we would be blaring everything all together and testing would be more difficult.
+
+So here are the Hardwar Output sections:
+
+Base Voicemeeter has Two Hardware Outputs:
+[
+
+
+
+Need Picture here
+
+
+
+]
+
+Voicemeeter Banana has Three Hardware Outputs:
+[
+
+
+
+Need Picture here
+
+
+
+]
+
+Voicemeeter Potato has Five Hardware Outputs:
+[
+
+
+
+Need Picture here
+
+
+
+]
+
+You will notice the A1-5 with a down arrow next to it. Upon clicking these you will be given a list of options you have available on your computer for outputting audio. This is important because you are able to send any source pulled into voice meeter out into each individual output which gets additional controls and adjustable options covered below:
+
+Base Voicemeeter Output Sliders and Settings:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Banana Ouput Sliders and Settings:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Potato Output Sliders and settings
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+You may notice new features being added going from Base Voicemeeter, to Voicemeeter Banana, then more in Voicemeeter Potato. Base Voicemeeter gets a Mix Down, Stereo REpeat, Composite toggle, mono toggle and mute toggle. Banana gains a mode selector, and EQ. Potato gains FX Returns.
```

Comment:
Would it be helpful to figure out a Markdown table to show the feature differences?

Reply:
Maybe, but lets take a look again at the new articles and see if they are necessary?